addons/website_event_exhibitor/views/event_templates_sponsor.xml

```diff
@@ -5,25 +5,17 @@
     <xpath expr="//div[@id='wrap']" position="inside">
         <div class="container mt32 mb16 d-none d-md-block d-print-none" t-if="event.sponsor_ids">
             <div t-attf-class="d-flex flex-wrap mb-5 #{'' if (len(event.sponsor_ids) > 10) else 'justify-content-md-center'}">
-                <t t-foreach="event.sponsor_ids" t-as="sponsor">
+                <t t-foreach="event.sponsor_ids.sorted(lambda sponsor: not sponsor.website_published and sponsor.exhibitor_type != 'sponsor')" t-as="sponsor">
+                    <t t-set="isSponsorUnpublished" t-value="not sponsor.website_published and sponsor.exhibitor_type != 'sponsor'"/>
```

Comment:
Variable name is misleading: it is not only linked to publish status. Also, we don't use camel case in templates :) .

Reply:
Renamed it :)